renv.lock

```diff
@@ -1,10 +1,10 @@
 {
   "R": {
-    "Version": "4.2.2",
+    "Version": "4.3.1",
     "Repositories": [
       {
         "Name": "CRAN",
-        "URL": "https://cran.rstudio.com"
+        "URL": "https://packagemanager.posit.co/cran/latest"
```

Comment:
Interesting - did it shift this itself, or is this something we should be doing with other repos too?

Reply:
It did that one itself